src/main/resources/messages/email.properties

```diff
@@ -1 +1,3 @@
-registration.success=Success! You have registered. Please click the link below to verify your email address. 
\ No newline at end of file
+registration.success=Success! You have registered. Please click the link below to verify your email address.
+password-reset.begin=Please click the link below to finish resetting your password.
+password-reset.success=You have successfully reset your Monuments and Memorials password.
```

Comment:
Should we make the email subjects messages too?

Reply:
We're going to have to redo this all to use HTML emails probably so I haven't bothered